website/content/en/docs/upgrading/compatibility.md

```diff
@@ -75,14 +75,13 @@ Karpenter offers three types of releases. This section explains the purpose of e
 
 ### Stable Releases
 
-Stable releases are the most reliable releases that are released with weekly cadence. Stable releases are our only recommended versions for production environments.
-Sometimes we skip a stable release because we find instability or problems that need to be fixed before having a stable release.
-Stable releases are tagged with a semantic version prefixed by a `v`. For example `v0.13.0`.
+Stable releases are the only recommended versions for production environments. Stable releases are tagged with a semantic version (e.g. `0.35.0`). Note that stable releases prior to `0.35.0` are prefixed with a `v` (e.g. `v0.34.0`).
 
 ### Release Candidates
 
-We consider having release candidates for major and important minor versions. Our release candidates are tagged like `vx.y.z-rc.0`, `vx.y.z-rc.1`. The release candidate will then graduate to `vx.y.z` as a normal stable release.
+We consider having release candidates for major and important minor versions. Our release candidates are tagged like `x.y.z-rc.0`, `x.y.z-rc.1`. The release candidate will then graduate to `x.y.z` as a normal stable release.
```

Comment:
```suggestion
We consider having release candidates for major and important minor versions. Our release candidates are tagged like `x.y.z-rc.0`, `x.y.z-rc.1`. The release candidate will then graduate to `x.y.z` as a normal, stable release.
```

nit

Reply:
Actually, I think this is correct, right? Since what is normal is the "stable release".

Reply:
Is there a non-normal stable?

Reply:
Commas imply synonym. No commas imply additional descriptors and variance

Reply:
I think the implication is that an RC is a non-normal stable release, that's how I read this at least.